src/new-client/testing/log-in.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect } from '@playwright/test';
+import env from './testEnvironment';
+
+const { TEST_USER_USERNAME, TEST_USER_PASSWORD } = env;
+
+test('Loads the site', async ({ page }) => {
```

Comment:
Could this be be asserted in the login test? i.e assert the page header before logging in.

Reply:
I'm not sure I quite see what you mean here, sorry. Could you clarify a bit? 

(I am happy however to remove this test as it doesn't really add value)

Reply:
In the sense of do we need a test to check that the site loads? Or could this assertion be part of the login test? i.e. before logging in check that the site has loaded. 

I guess I am coming form the mindset of E2E tests generally being more user journey specific so I would see this as the start of a user journey for example.

Reply:
Yeah, I think I only added it from a "fail fast" mentality whilst working (i.e. if this test fails, we can stop right away because everything else is guaranteed to fail). 
How about I just remove it entirely?

Reply:
As discussed on our call, have removed :)